### DISCUSSION TITLE
New chess variants

### DISCUSSION THREAD

**Discussion by Renifero** [gbtami/pychess-variants#1310] [Ideas]
Below is a link to my chess variants page. If anyone likes any of these variants then it can be implemented.

https://www.chessvariants.com/index/mainquery.php?type=Any&orderby=Type&displayauthor=1&displayinventor=1&authorid=ReniferOI&usethisheading=Items+Authored+by+Piotr++Smagacz

**Comment by gbtami** [comment]
Thx.

**Comment by gbtami** [comment]
Are these variants supported by Fairy-Stockfish?
Please read my general answer about adding new variants here https://discord.com/channels/634298688663191582/634298688663191584/1226913584521674792
